chromium_src/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/color/brave_color_id.h"
+
+#include "chrome/browser/ui/views/download/bubble/download_toolbar_button_view.h"
+
+#define DownloadToolbarButtonView DownloadToolbarButtonViewChromium
+
+#include "src/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc"
+
+#undef DownloadToolbarButtonView
+
```

Comment:
I'm not sure if it's worth, but can we remove the patch file by overriding gfx::RectFToSkRect() like this?

```suggestion
#include "ui/gfx/geometry/skia_conversions.h"

namespace gfx {
SkRect AdjustRingBounds(const gfx::RectF& ring_bounds);
}

#define RectFToSkRect(ring_bounds) AdjustRingBounds(ring_bounds)
#define DownloadToolbarButtonView DownloadToolbarButtonViewChromium

#include "src/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc"

#undef DownloadToolbarButtonView
#undef RectFToSkRect

namespace gfx {

SkRect AdjustRingBounds(const gfx::RectF& ring_bounds) {
  const int chromium_ring_radius = ui::TouchUiController::Get()->touch_ui()
                        ? kProgressRingRadiusTouchMode
                        : kProgressRingRadius;
  constexpr auto kBraveRingRadius = 12;
  const auto delta = kBraveRingRadius - chromium_ring_radius;
  gfx::RectF new_bounds(ring_bounds);
  new_bounds.Outset(gfx::OutsetsF(delta));
  return gfx::RectFToSkRect(new_bounds);
}

}  // namespace gfx
```